functions/src/definitions/api/handlers/patchVoteRequest.ts

```diff
@@ -0,0 +1,60 @@
+import { Request, Response } from "express"
+import { updateVoteRequest } from "../interfaces"
+import { Timestamp } from "firebase-admin/firestore"
+import * as admin from "firebase-admin"
+
+if (!admin.apps.length) {
+  admin.initializeApp()
+}
+
+const db = admin.firestore()
+
+const patchVoteRequestHandler = async (req: Request, res: Response) => {
+  // get message ID and voteRequestId
+  const messageId = req.params.messageId
+  const voteRequestId = req.params.voteRequestId
+  // check that both are passed
+  if (!messageId || !voteRequestId) {
+    return res.status(400).send("Message Id or vote request Id missing.")
+  }
+  //confirm category in body
+  const { category, truthScore } = req.body as updateVoteRequest
+  if (!category) {
+    return res.status(400).send("A category is required in the body")
+  }
+
+  if (category === "info" && !truthScore) {
+    return res.status(400).send("truthScore is required for info category")
+  }
+
+  if (truthScore) {
```

Comment:
```suggestion
  if (truthScore > 0) {
```
- Explicit check that conforms to the test rather than truthy-falsy check which is error-prone

Reply:
Amended this to combine category !== "info" condition as well